src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx

```diff
@@ -918,7 +919,25 @@ class EstimateYourBenefitsForm extends React.Component {
   );
 
   renderAboutYourSchool = () => {
-    if (!this.props.displayedInputs) return null;
+    const {
+      inState,
+      tuition,
+      books,
+      calendar,
+      enrolled,
+      enrolledOld,
+    } = this.props.displayedInputs;
+
+    if (
+      !inState &&
+      !tuition &&
+      !books &&
+      !calendar &&
+      !enrolled &&
+      !enrolledOld
```

Comment:
Would it be better to if(!(inState || tuition || books || calendar || enrolled || enrolledOld))?

Reply:
changed